src/Core/Resolvers/SqlQueryEngine.cs

```diff
@@ -61,7 +61,8 @@ public class SqlQueryEngine : IQueryEngine
         /// </summary>
         /// <param name="context">HotChocolate Request Pipeline context containing request metadata</param>
         /// <param name="parameters">GraphQL Query Parameters from schema retrieved from ResolverMiddleware.GetParametersFromSchemaAndQueryFields()</param>
-        public async Task<Tuple<JsonDocument?, IMetadata?>> ExecuteAsync(IMiddlewareContext context, IDictionary<string, object?> parameters)
+        /// <param name="dataSourceName">Name of datasource for which to set access token. Default dbName taken from config if null</param>
```

Comment:
Can this be null or simply an empty string?

Reply:
Ive default made it empty. It should not be null. in the future pr this will always be passed in from the callers of queryExecutor who need to be aware of the database to execute the query against.

Reply:
fixed.